commands/audit/scarunner.go

```diff
@@ -220,17 +235,21 @@ func GetTechDependencyTree(params xrayutils.AuditParams, tech coreutils.Technolo
 	default:
 		err = errorutils.CheckErrorf("%s is currently not supported", string(tech))
 	}
-	if err != nil || len(uniqueDeps) == 0 {
+	if err != nil || (len(uniqueDeps) == 0 && uniqDepsWithTypes == nil) {
```

Comment:
```suggestion
	if err != nil || (len(uniqueDeps) == 0 && len(uniqDepsWithTypes) == 0) {
```
len also check for nil, if its empty no need to continue as well right?